src/transformers/modeling_utils.py

```diff
@@ -87,6 +90,12 @@
 
 _init_weights = True
 
+if is_sagemaker_mp_enabled():
+    import smdistributed.modelparallel.torch as smp
+    from smdistributed.modelparallel import __version__ as SMP_VERSION
+
+    SMP_POST_1_10 = version.parse(SMP_VERSION) >= version.parse("1.10")
```

Comment:
```suggestion
    IS_SAGEMAKER_MP_POST_1_10 = version.parse(SMP_VERSION) >= version.parse("1.10")
else:
    IS_SAGEMAKER_MP_POST_1_10 = False
```
Let's put a name that has Sagemaker inside so a person reading the code later can understand what this is about.

Reply:
Sounds good to me!